internal/templates/port/template.go

```diff
@@ -0,0 +1,39 @@
+package port
+
+import (
+	"fmt"
+
+	"golang.stackrox.io/kube-linter/internal/check"
+	"golang.stackrox.io/kube-linter/internal/diagnostic"
+	"golang.stackrox.io/kube-linter/internal/objectkinds"
+	"golang.stackrox.io/kube-linter/internal/templates"
+	"golang.stackrox.io/kube-linter/internal/templates/port/internal/params"
+	"golang.stackrox.io/kube-linter/internal/templates/util"
+	v1 "k8s.io/api/core/v1"
+)
+
+func init() {
+	templates.Register(check.Template{
+		HumanName:   "ports",
+		Key:         "ports",
+		Description: "Flag containers exposing ports which not recommended as per security best practice",
```

Comment:
maybe change description to: `Flag containers exposing ports under protocols that match the supplied parameters`